src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  box-sizing: border-box;
```

Comment:
`box-sizing` is not inherited and body don't have any paddings or borders by default
this rule is innecessary here